examples/tv-casting-app/tv-casting-common/core/CastingPlayer.h

```diff
@@ -0,0 +1,168 @@
+/*
+ *
+ *    Copyright (c) 2023 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+#pragma once
+
+#include "Types.h"
+#include "lib/support/logging/CHIPLogging.h"
+
+#include <inet/IPAddress.h>
+#include <inet/InetInterface.h>
+#include <string.h>
+#include <vector>
+
+namespace matter {
+namespace casting {
+namespace core {
+
+enum ConnectionError
+{
+    NO_CONNECTION_ERROR = 0,
+    FAILED_TO_CONNECT   = 1
+};
+
+using ConnectCallback    = std::function<void(ConnectionError)>;
+using DisconnectCallback = std::function<void(void)>;
+
+const int kPortMaxLength = 5; // port is uint16_t
+const int kIdMaxLength   = chip::Dnssd::kHostNameMaxLength + kPortMaxLength;
+
+class Attributes
```

Comment:
consider a more descriptive name, like DiscoveredPlayerAttributes

Reply:
Made it more descriptive. Chose to name it "CastingPlayerAttributes"